packages/core/src/Stores/client-store.js

```diff
@@ -1181,6 +1181,7 @@ export default class ClientStore extends BaseStore {
         } else {
             Cookies.remove('region', { domain });
             Cookies.remove('client_information', { domain });
+            Cookies.remove('is_p2p_disabled', { domain });
```

Comment:
LGTM!
Can put it here as well, as another alternative.
![image](https://user-images.githubusercontent.com/125247833/230586637-55869371-9f9a-4424-8025-d5d42dfde38b.png)